app/services/email_masker.rb

```diff
@@ -3,19 +3,17 @@
 class EmailMasker
   def self.mask(email)
     email.gsub(/^(.+)@(.+)$/) do |_match|
-      local_part = $1
+      username_part = $1
```

Comment:
I am curious--why was the name changed from `local` to `username`?

Reply:
The ticket said username portion (admittedly I wrote the ticket from Anthony's comment) and I went from there. I can revert to `local` if you feel that it would be better.